packages/form-js-viewer/src/render/components/form-fields/Textfield.js

```diff
@@ -60,10 +61,11 @@ export default function Textfield(props) {
         id={ prefixId(id, formId) }
         onInput={ onChange }
         type="text"
-        value={ value } />
+        value={ value }
+        aria-describedby={ errorMessageId } />
     </InputAdorner>
     <Description description={ description } />
-    <Errors errors={ errors } />
+    <Errors errors={ errors } id={ errorMessageId } />
```

Comment:
We should add a [a11y test case](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-viewer/test/spec/render/components/form-fields/Textfield.spec.js#L249) where we render an error and check that everything works as expected.

```js
it('should have no violations', async function() {

  // given
  this.timeout(5000);

  const { container } = createTextfield({
    value: 'John Doe Company',
    errors: [ 'Something went wrong' ]
  });

  // then
  await expectNoViolations(container);
});
```

Not necessarily for each form field, but at least a few would make a lot of sense.

Reply:
Should be done now